this is a very abandoned python module, easier to just patch in this
diff --git a/buildtools/config.py b/buildtools/config.py
index 20a2fd9d..fa16befd 100644
--- a/buildtools/config.py
+++ b/buildtools/config.py
@@ -990,7 +990,11 @@
     if MSVCinfo is not None:
         return MSVCinfo
 
-    from attrdict import AttrDict
+    class AttrDict(dict):
+        def __getattr__(self, attr):
+            return self[attr]
+        def __setattr__(self, attr, value):
+            self[attr] = value
 
     # Note that it starts with a monkey-patch in setuptools.msvc to
     # workaround this issue: pypa/setuptools#1902
